lib/ui/playground/data/playground_atom_datasource.dart

```diff
@@ -163,6 +166,30 @@ class PlayGroundAtomDataSource implements PlaygroundDataSource {
             viewModel: MockStageCardViewModel.buildCompleteState(),
           ),
         ),
+      ),
+      PlaygroundWidget(
```

Comment:
could you nest these into one entry, @DavidFD005 can demo how.

Reply:
Done

Reply:
Done